### DISCUSSION TITLE
Middleware with minimal api with .net 6.0

### DISCUSSION THREAD

**Discussion by misecca** [dotnet/AspNetCore.Docs#29247] [Q&A]
Hi all, I've follow the instruction described here: [Middleware in Minimal API apps](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/middleware?view=aspnetcore-7.0)

But with .net 6.0 doesn't works.
 This is because the interaction "Middleware"-"minimal api" works only starting from .net 7.0 ?

**Comment by Rick-Anderson** [comment]
Yes
